acre.yaml

```diff
@@ -596,6 +619,8 @@ validation:
   dashboard: (( defined( landscape.dashboard ) ? validate( landscape.dashboard, validation.dashboard_validator, validation.dashboard_no_seed_strategy_validator, [validation.dashboard_terminals_validator, landscape.cert-manager] ) :~~ ))
   ##### landscape.certmanager
   cert-manager: (( validate( landscape.cert-manager, validation.cert-manager_validators.basic, [validation.cert-manager_validators.email, landscape.identity] ) ))
+  ##### landscape.extensionsValuesOverwrite
+  extensionsValuesOverwrite: (( defined( landscape.extensionsValuesOverwrite ) ? validate( landscape.extensionsValuesOverwrite, validation.extensionsValuesOverwrite_validator ) :~~ ))
```

Comment:
```
extensionsValuesOverwrite: (( defined( landscape.extensionsValuesOverwrite ) ? validate( keys( landscape.extensionsValuesOverwrite ), ["list", ["valueset", keys( landscape.versions.gardener.extensions )]] ) :~~ ))
```
What this does: It takes the keys from the `landscape.extensionsValuesOverwrite` map and validates that this is a list (always true, since the `keys` function returns a list) and that each entry of that list matches one of the entries of `keys( landscape.versions.gardener.extensions )` (which basically lists all the extensions).

Reply:
The provided expressions needs to be adapted if we move the configuration to `landscape.gardener.extensions.<extension_name>.valueOverwrites`.

Reply:
Thanks for the validation. I changed the validation for that and moved it to `landscape.gardener.extension´